frontend/src/components/admin/DocumentUploadsMetabaseDashboard.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from "react";
+import { AuthorizationGuard } from "@/HOC/AuthorizationGuard";
+import * as Permission from "@/constants/permissions";
+
+/**
+ * @class DocumentUploadsMetabaseDashboard displays an iframe housing Document Upload information.
+ */
+
+const jwt = require("jsonwebtoken");
+
+const METABASE_SITE_URL = "https://mds-metabase-empr-mds-prod.pathfinder.gov.bc.ca";
+const METABASE_SECRET_KEY = "623e8cbb9f381e40651494194032c005ba8e92559ba291a8587d1b8e021cb454";
+
+const payload = {
+  resource: { dashboard: 1 },
+  params: {}
+};
+const token = jwt.sign(payload, METABASE_SECRET_KEY);
```

Comment:
Looking at how we're handling token here, it would probably be better for this to be in the component state. It could be fetched on one of the component lifecycle hooks and added to the state when the token is returned. Then referenced as `this.state.token` inside of the component

Reply:
@NWCalvank If you want this moved into the state, then the above comment to convert `DocumentUploadsMetabaseDashboard.js` to a pure function is no longer needed?

Reply:
Yes, they were two, independent suggestions that depended on which solution was taken.